### PR TITLE
New table Address

### PR DESCRIPTION
**Pull request by cheppymk**
New migration included
Pls update the database ASAP

### CODE REVIEW

File: Shipfinity.Domain/Models/Order.cs [hunk 1]
```diff
@@ -10,7 +11,10 @@ namespace Shipfinity.Domain.Models
     public class Order : BaseEntity
     {
         public Customer Customer { get; set; }
+        [Required]
         public int CustomerId { get; set; }
+        public Address Address { get; set; }
+        public int AddressId { get; set; }
```
Comment (KristijanJankuloski) [inline Shipfinity.Domain/Models/Order.cs:17]: AddressId should also be required here

File: Shipfinity.Domain/Models/Address.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shipfinity.Domain.Models
+{
+    public class Address : BaseEntity
+    {
+        public string AddressLine { get; set; }
```
Comment (KristijanJankuloski) [inline Shipfinity.Domain/Models/Address.cs:12]: Limit the characters of these string values so they are not created as varchar(MAX) example Zipcode should be fairy short MaxLength(10) maybe or 15, AddressLine should be around MaxLegnth(100) so on

File: Shipfinity.Api/appsettings.Development.json
Comment (KristijanJankuloski) [inline Shipfinity.Api/appsettings.Development.json:1]: Revert changes to this app settings

**Review by KristijanJankuloski — CHANGES_REQUESTED** [review]

File: Shipfinity.Domain/Models/Address.cs (hunk 2 above)
Comment (cheppymk) [reply Shipfinity.Domain/Models/Address.cs:14]: done

File: Shipfinity.Api/appsettings.Development.json
Comment (cheppymk) [reply Shipfinity.Api/appsettings.Development.json:1]: done

**Review by KristijanJankuloski — APPROVED** [review]

**Review by Aleksandar1105 — APPROVED** [review]

**Pull request merged**